src/assets/scripts/game/GameController.js

```diff
@@ -1,13 +1,35 @@
 /* eslint-disable camelcase, no-underscore-dangle, no-mixed-operators, func-names, object-shorthand,
 no-undef, class-methods-use-this */
 import $ from 'jquery';
+import _forEach from 'lodash/forEach';
 import GameOptions from './GameOptions';
 import { round } from '../math/core';
 import { SELECTORS } from '../constants/selectors';
+import { TIME } from '../constants/globalConstants';
 
 // Temporary const declaration here to attach to the window AND use as internal property
 const game = {};
 
+// TODO: Remember to move me to wherever the constants end up being moved to
+/**
+ * Definitions of point values for given score events
+ * @type {Object}
+ */
+const GAME_EVENTS = {
```

Comment:
@n8rzz You'll see I have removed the export here, and the imports to other files. After actually using my brain, I realized that importing `GAME_EVENTS`, and trying to say `window.gameController.record_game_event(GAME_EVENTS.ARRIVAL);` because `GAME_EVENTS.ARRIVAL` has a value of `10`, which is of no use when you try to add the event. I've instead just fed `'ARRIVAL'`, in each one. Not sure if there's a better way to do it, but that's where I'm at with it now.